### ISSUE TITLE
Winmerge 2011 Linefilter doesn't work

### ISSUE THREAD

**Issue by urbantamas** [datadiode/winmerge2011#20]
It seems that line filter doesn't work for a while in Winmerge 2011. I'm installe the latest release, which is 0.2011.211.182. 

I've some Oracle APEX application export files (PL/SQL) where I want to filter out those lines which contains a "=>wwv_flow_api.id(xxx)" call anywhere in the current file. xxx always means an integer number. 

I set a regular expression in the Tools|Filters|Linefilter and enabled it also. But after refreshing the compare nothing has changed at all. If I select this kind of line in the compare window and also select the referenced line filter the test case will underscore the whole line with green color (which should means the regexp search was success?)

I appreciate some help in this case. Thanks.

**Comment by datadiode** [comment]
I cannot confirm this issue. Regexp "=>wwv_flow_api.id\(\d+\)" seems to yield expected results.
Since it might depend on settings, can you share the outcome of _Help_ &rarr; _Configuration_?

**Comment by urbantamas** [comment]
![image](https://user-images.githubusercontent.com/47850488/137628224-5ff9f18f-fff2-4b97-8c87-67313e0be911.png)

**Comment by urbantamas** [comment]
Yes, when I cleared a couple of line filter options, the comparison ignored the id value differences between the rows what contains the "wwv_flow_api.id()" reference

[page_00202_SIDEA.sql.txt](https://github.com/datadiode/winmerge2011/files/7704495/page_00202_SIDEA.sql.txt)
[page_00202_SIDEB.sql.txt](https://github.com/datadiode/winmerge2011/files/7704496/page_00202_SIDEB.sql.txt)
.

**Comment by urbantamas** [comment]
do you have any update for this issue? thanks.

**Comment by urbantamas** [comment]
Can you give me some instructions how to solve this issue?

**Comment by datadiode** [comment]
Can you share your LineFilters.ini?
Although I lost focus on this project, I'd like to understand and fix the bug.

**Comment by urbantamas** [comment]
Where are the exact locations of the configuration files? The location used under Option|System|Supplement Folder points to my My Documents\Winmerge folder. But the LineFilters.ini does not contain my custom line filters I set.

**Comment by urbantamas** [comment]
[LineFilters másolata.ini.txt](https://github.com/datadiode/winmerge2011/files/9850674/LineFilters.masolata.ini.txt)

**Comment by datadiode** [comment]
Oops, I didn't remember that line filters entered through GUI are stored in the Windows registry.
Can you export them to a file?
![image](https://user-images.githubusercontent.com/10423465/197544761-22747ca7-cedd-4f10-a008-ccc0ede2ea5d.png)

**Comment by urbantamas** [comment]
[winmerge.2011.linefilters.reg.txt](https://github.com/datadiode/winmerge2011/files/9859005/winmerge.2011.linefilters.reg.txt)
